adapters/src/main/java/io/realm/RealmBaseAdapter.java

```diff
@@ -34,47 +32,17 @@
  */
 public abstract class RealmBaseAdapter<T extends RealmModel> extends BaseAdapter {
     @Nullable
-    @Deprecated
-    protected LayoutInflater inflater;
-    @Nullable
     protected OrderedRealmCollection<T> adapterData;
-    @Nullable
-    @Deprecated
-    protected Context context;
-    private final RealmChangeListener<? extends BaseRealm> listener;
-
-    @Deprecated
-    public RealmBaseAdapter(@NonNull Context context, @Nullable OrderedRealmCollection<T> data) {
-        if (data != null && !data.isManaged())
-            throw new IllegalStateException("Only use this adapter with managed list, " +
-                    "for un-managed lists you can just use the BaseAdapter");
-        //noinspection ConstantConditions
-        if (context == null) {
-            throw new IllegalArgumentException("Context cannot be null");
-        }
-        this.context = context;
-        this.adapterData = data;
-        this.inflater = LayoutInflater.from(context);
-        this.listener = new RealmChangeListener<BaseRealm>() {
-            @Override
-            public void onChange(BaseRealm results) {
-                notifyDataSetChanged();
-            }
-        };
-
-        if (data != null) {
-            addListener(data);
-        }
-    }
+    private final RealmChangeListener<Object> listener;
 
     public RealmBaseAdapter(@Nullable OrderedRealmCollection<T> data) {
         if (data != null && !data.isManaged())
             throw new IllegalStateException("Only use this adapter with managed list, " +
                     "for un-managed lists you can just use the BaseAdapter");
         this.adapterData = data;
-        this.listener = new RealmChangeListener<BaseRealm>() {
+        this.listener = new RealmChangeListener<Object>() {
```

Comment:
Agree. I think it's OK to use raw type (`RealmChangeListener`). 
`RealmRecyclerViewAdapter` also is using raw type.